docs/design/software-design-specification.adoc

```diff
@@ -262,7 +262,13 @@ Input parameters:
 
 | SDS-TASK-18
 | `build-typescript.sh` shell script
-a| Runs `npm run build`, and copies the folder specified in `build-dir` into the directory specified by `output-dir`.
+a| Checks that package.json and package-lock.json exist to require best practice of using lock files. See also https://github.com/opendevstack/ods-pipeline/discussions/411
+
+Runs `npm run build`, and copies the folder specified in `build-dir` into directory `dist` which is placed into the directory specified by `output-dir`.
```

Comment:
Maybe change to:
*... and copies **the files inside** the folder specified in `build-dir`...*

What do you think?

Reply:
Makes sense!

Reply:
I now updated the spec.